### PR TITLE
CiscoS300 'show interfaces description' template

### PR DESCRIPTION
**Pull request by PavloSkliarenko**
Template + test

### CODE REVIEW

File: ntc_templates/templates/cisco_s300_show_interfaces_description.textfsm [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Value PORT (\S+)
```
Comment (jvanderaa) [inline ntc_templates/templates/cisco_s300_show_interfaces_description.textfsm:1]:
```suggestion
Value INTERFACE (\S+)
```

**Review by jvanderaa — CHANGES_REQUESTED** [review]
Looking to update to the data model for interfaces.

**Comment by PavloSkliarenko** [comment]
How can I re-run test, because they failed because of this [PR](https://github.com/networktocode/ntc-templates/pull/1557)?

**Comment by PavloSkliarenko** [comment]
Fix value names according to new Data model

**Review by jvanderaa — APPROVED** [review]

**Pull request merged**